extensions/blocks/podcast-player/style.scss

```diff
@@ -261,8 +261,9 @@ $player-background: transparent;
 		fill: var( --jetpack-podcast-player-primary );
 
 		svg {
-			position: absolute;
+			display: block;
 			width: $track-status-icon-size;
+			height: $track-status-icon-size;
```

Comment:
Can we also make sure the `$track-status-icon-size` is 24px? It would be consistent with other icons in Gutenberg (24px is the default size).

Reply:
I changed it to 24px to test, and that makes the playing track 1px taller than the other tracks. I'll add an issue for us to revisit this structure.

Reply:
Tracked https://github.com/Automattic/jetpack/issues/15373